stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/StringUtils.kt

```diff
@@ -4,7 +4,7 @@ internal object StringUtils {
 
     fun removeTimePrefix(attachmentName: String?, usedDateFormat: String): String? {
         val dataFormatSize = usedDateFormat.length + 1
-        val regex = "^prefix_\\S{$dataFormatSize}".toRegex()
+        val regex = "^STREAM_\\S{$dataFormatSize}".toRegex()
```

Comment:
I think we should move it to constant because we are also using it in `StorageHelper`

Reply:
👍